src/index.html

```diff
@@ -22,6 +22,391 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="Moyo header"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
```

Comment:
The `data-qa` attribute here should be reviewed to ensure it aligns with the task requirements. Ensure that the attribute values are unique and meaningful for testing purposes.